embedded-influxdb/pom.xml

```diff
@@ -2,12 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <artifactId>testcontainers-spring-boot</artifactId>
         <groupId>com.playtika.testcontainers</groupId>
-        <version>1.40-SNAPSHOT</version>
+        <version>1.43-SNAPSHOT</version>
```

Comment:
hmm, good that you notices, but in order to fix it properly module should be added to https://github.com/testcontainers/testcontainers-spring-boot/blob/develop/pom.xml also, so next release will change this value